app.config.yaml

```diff
@@ -13,8 +13,13 @@ application:
           certPassword: $CERT_PASSWORD
           certKey: $CERT_KEY
           certThumbprint: $CERT_THUMB_PRINT
+          enablePreviewPublish: $ENABLE_PREVEW_PUBLISH
           groupCheckUrl: $GROUP_CHECK_URL
-          grayboxUserGroups: $GRAYBOX_USER_GROUPS     
+          grayboxUserGroups: $GRAYBOX_USER_GROUPS
+          helixAdminApiKeys: $HELIX_ADMIN_API_KEYS
+          bulkPreviewCheckInterval: $BULK_PREVIEW_CHECK_INTERVAL
+          maxBulkPreviewChecks: $MAX_BULK_PREVIEW_CHECKS
+          skipInProgressCheck: $SKIP_INPROGRESS_CHECK
```

Comment:
This is not getting used yet -> `skipInProgressCheck`

Reply:
Done. Thanks